clang/lib/Parse/Parser.cpp

```diff
@@ -1255,8 +1256,12 @@ Parser::DeclGroupPtrTy Parser::ParseDeclarationOrFunctionDefinition(
   // Add an enclosing time trace scope for a bunch of small scopes with
   // "EvaluateAsConstExpr".
   llvm::TimeTraceScope TimeScope("ParseDeclarationOrFunctionDefinition", [&]() {
-    return Tok.getLocation().printToString(
-        Actions.getASTContext().getSourceManager());
+    llvm::TimeTraceMetadata M;
```

Comment:
The column might be a little redundant for functions, but may be useful for other events to help identify them.
Should we add it now? Are there any downsides to adding it?

Reply:
Added column information as well. The source range (begin and end column) might also be interesting for EvaluateExpr* thing but I am omitting them as of now.